pkg/client/actionclient.go

```diff
@@ -249,6 +250,10 @@ func (c *actionClient) Upgrade(name, namespace string, chrt *chart.Chart, vals m
 	return rel, nil
 }
 
+func (c *actionClient) Rollback(name string, opts ...RollbackOption) error {
+	return c.rollback(name, concat(c.upgradeFailureRollbackOpts, opts...)...)
```

Comment:
This is interesting. For `Uninstall`, we have separate `uninstallOpts` and `installFailureUninstallOpts`. We probably need to do the same thing here?

Reply:
Updated, thanks!